static/js/input.js

```diff
@@ -362,8 +362,8 @@ function parseFeaColumn(arr) {
  * @todo Terminate the search for categories and features if the entropy of
```

Comment:
This probably isn't necessary any more. Previously I was thinking that the program will read the file line by line. But it seems that a client-end webpage cannot do that. Therefore we have to read the entire file then start to process.